components/Downloads/DownloadHeader/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { FormattedMessage, useIntl } from 'react-intl';
+import styles from './index.module.scss';
+import SectionTitle from '../../Common/SectionTitle';
+import type { NodeRelease } from '../../../types/releases';
+import type { FC } from 'react';
+
+type DownloadHeaderProps = {
+  release: NodeRelease;
+};
+
+const DownloadHeader: FC<DownloadHeaderProps> = ({ release }) => {
```

Comment:
```suggestion
const DownloadHeader: FC<NodeRelease> = ({ versionWithPrefix, isLts, npm }) => {
```
Don't forget to let prettier fix the formatting :)